R/readUN_PopDiv.R

```diff
@@ -3,19 +3,29 @@
 #' Read-in UN population data.
 #'
 #' @param subtype Either "pop" or "lab".
-#' @param subset Either "estimates" or "medium".
+#' @param subset Either "estimates", "medium" or "1900-2150" for full time range.
 #' @inherit madrat::readSource return
 #' @seealso [madrat::readSource()] and [madrat::downloadSource()]
+#' @author Johannes Koch (& Merlin Jo Hosak for 1900-2150 subset)
 #' @order 2
 readUN_PopDiv <- function(subtype, subset = "estimates") { # nolint: object_name_linter.
   # Check function input
   if (!subtype %in% c("pop", "lab")) {
     stop("Bad input for readUN_PopDiv. Invalid 'subtype' argument.")
   }
-  if (!subset %in% c("estimates", "medium")) {
+  if (!subset %in% c("estimates", "medium", "1900-2150")) {
     stop("Bad input for readUN_PopDiv. Invalid 'subset' argument.")
   }
 
+  if (subtype == "pop" && subset == "1900-2150") {
+    file <- "UN_World_Population_1900-2150.csv"
+    x <- utils::read.csv(file)
+    x <- as.magpie(x)
+    x <- x * 1e9 # Convert from billions to individuals
```

Comment:
I also prefer to avoid end of line comments... can you place the comment in the line above?